doc/external_resources.md

```diff
@@ -12,6 +12,16 @@ to submit a pull request with your recommended addition to the
 
 :::::{grid} 1 2 2 3
 
+::::{grid-item-card} 2024 AGU PREWS9: Mastering Geospatial Visualizations with GMT/PyGMT
+:link: https://github.com/GenericMappingTools/agu24workshop
+:text-align: center
+:margin: 0 3 0 0
+
+![](https://github.com/user-attachments/assets/f92e1391-78f4-41bc-8ed8-5df1f59b8362)
```

Comment:
Could we have a slightly taller screenshot?

```suggestion
![](https://github.com/user-attachments/assets/9f3ab0ed-83f1-4cff-bc68-387fc13ca61f)
```

![image](https://github.com/user-attachments/assets/9f3ab0ed-83f1-4cff-bc68-387fc13ca61f)

Reply:
I also prefer to show the figure of Tutorial 4 completely, but there is already quite a lot white space when using the current screenshot in the neighboring cards, so I was unsure using a taller screenshot. But let's try and see how it looks.

Reply:
What about this one? The title is removed from the screenshot.
![Screenshot from 2024-12-21 12-12-25](https://github.com/user-attachments/assets/b86fd4c1-c5dd-4d51-a7fe-362d86cc01b0)

`https://github.com/user-attachments/assets/b86fd4c1-c5dd-4d51-a7fe-362d86cc01b0`

Reply:
Actually, I thought about not including the title in the screenshot, but kept it to be consistent with the image for the EGU2022 workshop. But let's see how it looks after https://github.com/GenericMappingTools/pygmt/pull/3689/commits/2e8b38ef1a0357470282628584d57982d878b8ca.

Reply:
I think you can choose anyone you like, then merge this PR.